modules/nlu/src/backend/pipelines/intents/svm_classifier.ts

```diff
@@ -62,9 +62,12 @@ export default class SVMClassifier {
       throw new Error('Could not find intents TFIDF model')
     }
 
-    const { l0Tfidf, l1Tfidf } = JSON.parse(tfidfModel.model.toString('utf8'))
+    const { l0Tfidf, l1Tfidf, token2vec } = JSON.parse(tfidfModel.model.toString('utf8'))
     this.l0Tfidf = l0Tfidf
     this.l1Tfidf = l1Tfidf
+    this.token2vec = token2vec
```

Comment:
from what I understand token2vec is another level of cache, am I correct ?

Reply:
yes